### DISCUSSION TITLE
How to follow build instructions for this project

### DISCUSSION THREAD

**Discussion by talhaYZ7** [Etar-Group/Etar-Calendar#1764] [Q&A] [answered]
I get errors when I try to build this project in Android Studio.

Can someone tell me step by step how to follow these build instructions?

**Comment by jspricke** [comment]
Can you share the exact error messages?

**Comment by talhaYZ7** [comment]
Here is the error message

> Namespace not specified. Specify a namespace in the module's build file: C:\Users\xyz\Desktop\Etar\Etar-Calendar\external\calendar\build.gradle. See https://d.android.com/r/tools/upgrade-assistant/set-namespace for information about setting the namespace.
> 
> If you've specified the package attribute in the source AndroidManifest.xml, you can use the AGP Upgrade Assistant to migrate to the namespace value in the build file. Refer to https://d.android.com/r/tools/upgrade-assistant/agp-upgrade-assistant for general information about using the AGP Upgrade Assistant.
>

**Comment by jspricke** [comment, marked as answer]
Did you clone the submodules as written here: https://github.com/Etar-Group/Etar-Calendar?tab=readme-ov-file#build-instructions
What is the output of  `git submodule status`?

**Reply by talhaYZ7** [reply]
I didn't clone the submodules. `git submodule status` output this

> -406e37bfd68c4707b3b8fa0bc795e6a3f7a156e4 external/calendar
> -e86358641aab03991bf3d56b36179f7773a83189 external/chips
> -404c08afb3b76681e3454942b71237ca76786613 external/colorpicker
> -82edadf36e0bdb0110cb19daa6bc3625802b2ac1 external/ex
> -03e2abd0d58263e26c3e5a26ed8ff76411b65960 external/timezonepicker

**Comment by jspricke** [comment]
Well, than it is expected to fail.

**Reply by talhaYZ7** [reply]
Thanks! I've been trying to fiqure this out since yesterday.
